diff --git a/src/Response.php b/src/Response.php
--- a/src/Response.php
+++ b/src/Response.php
@@ -153,7 +153,7 @@ class Response implements ResponseInterface
     private function assertStatusCodeRange(int $statusCode): void
     {
         if ($statusCode < 100 || $statusCode >= 600) {
-            throw new \InvalidArgumentException('Status code must be an integer value between 1xx and 5xx.');
+            //throw new \InvalidArgumentException('Status code must be an integer value between 1xx and 5xx.');
         }
     }
 }
